android/src/main/java/dev/mcodex/RNSensitiveInfo/RNSensitiveInfoModule.java

```diff
@@ -159,7 +159,13 @@ public void setInvalidatedByBiometricEnrollment(final boolean invalidatedByBiome
 
     @ReactMethod
     public void hasEnrolledFingerprints(final Promise pm) {
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M && mFingerprintManager != null) {
+        if (mFingerprintManager == null) {
+            pm.resolve(false);
+        }
+
+        if (Build.VERSION.SDK_INT == Build.VERSION_CODES.M) {
```

Comment:
Should this have been an else if or cam this resolve multiple times?

Reply:
Not sure what you mean. The intention of this code is to only run the first block on Android 6 devices, and run the next one on versions that are above 6, otherwise reject (probably since Android version is below 6 which technically shouldn't happen.)

Reply:
what i mean is that there is an if statement above this, which could be true, as well as this if statement, resulting in resolve being called twice